tests/conftest.py

```diff
@@ -76,3 +77,21 @@ def initiate_empty_pepdb_con(
     pepdb_con = PEPDatabaseAgent(dsn=DNS, echo=False)
 
     yield pepdb_con
+
+
+@pytest.fixture(scope="function")
+def create_groups(
+    list_of_available_peps,
+):
+    """
+    create connection without adding peps to the db
+    """
+    # sa_engine = create_engine(DNS)
+    # with sa_engine.begin() as conn:
+    #     conn.execute(text("DROP table IF EXISTS projects CASCADE"))
+    #     conn.execute(text("DROP table IF EXISTS samples CASCADE"))
+    #     conn.execute(text("DROP table IF EXISTS subsamples CASCADE"))
+
+    pepdb_con = PEPDatabaseAgent(dsn=DNS, echo=False)
+
+    yield pepdb_con
```

Comment:
this code doesn't make sense to me. the name of the function, docstring, and code don't match.